lib/iex/cloud/request.rb

```diff
@@ -21,13 +21,14 @@ def delete(path, options = {})
 
       def request(method, path, options)
         path = [endpoint, path].join('/')
+        options[:token] ||= publishable_token
```

Comment:
With this change, If you call an API that requires a secret token and you haven't specified one, it's getting the publishable token. So you get back an error saying you have used the wrong token, as opposed to you have not specified a token. Which is wrong.